### ISSUE TITLE
Glass Textures

### ISSUE THREAD

**Issue by VaguelyPro** [Chocohead/OptiFabric#1036]
Connected glass textures are not correct. There are lines on certain sides of the glass. This is not an optifine issue, this only happens when optifabric is installed
![Screenshot_SEF1](https://user-images.githubusercontent.com/131079491/232632911-8a636172-2a4d-48f8-87c5-b548a1378799.png)

**Comment by TheCrafter8000** [comment]
Same here

**Comment by RU-B0** [comment]
Just wanted to comment how I fixed this issue: I found out that a mod called fusion (https://modrinth.com/mod/fusion-connected-textures) launches with OptiFine just fine. Using Midnighttigger's Default Connected Textures (https://modrinth.com/resourcepack/mt-ct-d/version/tfKvt8nl) and putting it as lowest priority on the resource pack list (but not below MC itself or fabric), it gets rid of this issue. Plus, OptiFine's implementation still works for other resource packs. Its a win - win. 

Tested with Fusion 1.0.5a and Optifine HD U I5, Optifabric 1.13.25, and FAPI 0.83.1 (+10000000000 other mods but you get the point)
